test/CoveragePool.test.js

```diff
@@ -61,20 +61,275 @@ describe("CoveragePool", () => {
     await assetPool.connect(underwriter).deposit(to1e18(400))
   })
 
+  describe("beginRiskManagerApproval", () => {
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        const notGovernance = await ethers.getSigner(5)
```

Comment:
Non-blocking, but we could abstract it as e.g. `thirdParty` and keep it next to `riskManager` and `underwriter`, just to make sure signer with index 5 is not going to be accidentally used in the future for some other role.